src/pybind/mgr/stats/fs/perf_stats.py

```diff
@@ -1,4 +1,5 @@
 import re
+import ipaddress
```

Comment:
maybe just use "from ipaddress import ip_address" since that's the only usage right now?

Reply:
Done